bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/bot/TelegramActions.java

```diff
@@ -341,6 +341,15 @@ public boolean sendTelegramPhoto(@ActionInput(name = "chatId") @Nullable Long ch
                 // Load image from provided base64 image
                 logger.debug("Photo base64 provided; converting to binary.");
                 try {
+                    if (photoURL.startsWith("data:")) // support data URI scheme
+                    {
+                        String[] photoURLParts = photoURL.split(",");
+                        if (photoURLParts.length > 1) {
+                            photoURL = photoURLParts[1];
+                        } else {
+                            logger.warn("The provided base64 string doesn't seem to a be valid data URI schema");
```

Comment:
"The provided base64 string is not a valid data URI scheme"

Reply:
fixed